Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1549,7 +1549,7 @@ void getSupportedTypes(HashSet<String>& types) const final
 
 void MediaPlayerPrivateWebM::registerMediaEngine(MediaEngineRegistrar registrar)
 {
-    if (!isAvailable())
+    if (!isAvailable() && !SourceBufferParserWebM::isAvailable())
```

Comment:
Why not changing MediaPlayerPrivateWebM::isAvailable instead?

Reply:
this one line is the actual fix for the bug. calling `SourceBufferParserWebM::isAvailable()` would have automatically linked libwebm